### PR TITLE
Update testkit.

### PR DESCRIPTION
**Pull request by olafurpg**
- Explain in changelog
- Run property tests to avoid regression

### CODE REVIEW

**Comment by xeno-by** [comment]
```
[error] (run-main-0) java.io.IOException: Cannot run program "wget": error=2, No such file or directory
java.io.IOException: Cannot run program "wget": error=2, No such file or directory
```

**Comment by olafurpg** [comment]
Hurray! Finally got this past CI : ) 

Seems the property test takes 80s to run, which I think it totally OK given it might catch tricky corner case bugs in pretty-printer and parser.

File: .drone.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ build:
       - COURSIER_CACHE=/drone/cache/coursier
     commands:
       - ./bin/checkCLA.sh
-      - sbt -Dsbt.ivy.home=/drone/cache/ivy2 ++$SCALA_VERSION test doc
+      - sbt -Dsbt.ivy.home=/drone/cache/ivy2 ++$SCALA_VERSION test doc "testkit/test:runMain  scala.meta.testkit.ScalametaParserPropertyTest"
```
Comment (xeno-by) [inline .drone.yml]: Do we need the double whitespace between `runMain` and `scala.meta...`?

File: .jvmopts [hunk 2]
```diff
@@ -1,4 +1,8 @@
--XX:-TieredCompilation
--Xss2m
--Xms725m
--Xmx725m
+-Xms4000m
+-Xmx55g
+-Xss4m
+-XX:MaxPermSize=2000m
```
Comment (xeno-by) [inline .jvmopts]: Why do we need the MaxPermSize option if .jvmopts already specify JVM 1.8?

File: changelog/1.5.0.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 ## `testkit`
 
-  * We have introduced `testkit`, an infrastructure to test metaprograms like a pro.
-    See [https://github.com/scalameta/scalameta/commit/506c284fd389fcbd12c47550a8527a9d051aa3b4](https://github.com/scalameta/scalameta/commit/506c284fd389fcbd12c47550a8527a9d051aa3b4) for more information.
+  * `testkit` is a new module that can be used to run property-based tests
```
Comment (xeno-by) [inline changelog/1.5.0.md]: What's a property-based test?

**Comment by xeno-by** [comment]
Agreed, the slowdown is totally justified. Apart from the minor comments above, LGTM.

File: .drone.yml (hunk 1 above)
Comment (olafurpg) [inline .drone.yml]: Done

File: .jvmopts (hunk 2 above)
Comment (olafurpg) [inline .jvmopts]: Good catch, I switched to a jdk 7 docker image in a temporary commit

Comment (olafurpg) [inline .jvmopts]: removed.

File: changelog/1.5.0.md (hunk 3 above)
Comment (olafurpg) [inline changelog/1.5.0.md]: [See here](http://letmegooglethat.com/?q=property+based+testing)

Comment (olafurpg) [inline changelog/1.5.0.md]: hehe :p sry, do you think it needs more explanation?

Comment (xeno-by) [inline changelog/1.5.0.md]: Just a URL should be enough, so that we're sure that people reading the notes understand what's going on :)

**Comment by xeno-by** [comment]
I'll merge the PR, because I'll be overhauling the build today. If you'd like to follow up on changing the changelog, please submit it as a separate PR. Thanks a lot for your work!

**Comment by olafurpg** [comment]
I'll update the changelog tomorrow!
